core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -532,6 +532,28 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
     }
   }
 
+  test("event queue size can be configued through spark conf") {
+    val conf = new SparkConf(false)
+      .set(LISTENER_BUS_EVENT_QUEUE_CAPACITY, 5)
+      .set("spark.scheduler.listenerbus.eventqueue.shared.capacity", "1")
+      .set("spark.scheduler.listenerbus.eventqueue.eventLog.capacity", "2")
+
+    val bus = new LiveListenerBus(conf)
+    val counter1 = new BasicJobCounter()
+    val counter2 = new BasicJobCounter()
+    val counter3 = new BasicJobCounter()
+
+    bus.addToSharedQueue(counter1)
```

Comment:
Maybe add comment to explain this is just to trigger add a new Queue.

Reply:
added comments